packages/design-system/src/components/contextMenu/components/subMenuTrigger.js

```diff
@@ -50,6 +50,8 @@ function SubMenuTrigger({
   isRTL = false,
```

Comment:
Looks like this component is not used for the submenu, are the changes here still needed?

Reply:
Good point, I ended up scrubbing using that one. Will revert the changes.

Reply:
Reverted in 43e08670cb.